docs/reference-guides/rancher-webhook.md

```diff
@@ -19,7 +19,7 @@ Each Rancher version is designed to be compatible with a single version of the w
 
 | Rancher Version | Webhook Version | Availability in Prime | Availability in Community |
 |-----------------|-----------------|-----------------------|---------------------------|
-| v2.8.3          |     v0.4.3      | &cross;               | &check;                   |
+| v2.8.3          |     v0.4.3      | &check;               | &check;                   |
```

Comment:
If it's not available on Prime, shouldn't this remain a cross?

Reply:
Typo in the description. `Not` should be `now`.